Tests/VsCodeExtension/Requests.fs

```diff
@@ -489,11 +489,9 @@ let initialize(solutionPath: string) =
     }}
 }}"
 
-let initialized() = @"{""jsonrpc"":""2.0"",""method"":""initialized"",""params"":{},""id"":10}}"
+let textDocumentDocumentSymbol = @"{ ""jsonrpc"":""2.0"", ""method"": ""textDocument/documentSymbol"", ""id"":1 }"
 
-let textDocumentDocumentSymbol() = @"{""jsonrpc"":""2.0"",""method"":""textDocument/documentSymbol"",""id"":1}"

```

Comment:
Can we reinstate master's version of `textDocumentDocumentSymbol` here? Seems like you didn't want to change that.